pkg/provider/register/register.go

```diff
@@ -17,6 +17,7 @@ package register
 // packages imported here are registered to the controller schema.
 // nolint:revive
 import (
+	_ "github.com/external-secrets/external-secrets/pkg/provider/akeyless"
```

Comment:
I don't think we are going to have a Akeyless dedicated account for this project, at least right now. I think we can left it out here like we don't have alibaba and gilatb.

Reply:
Sorry, I was sleepy, I meant in e2e/suite/import.go, you are right :sweat_smile: